src/flows/parser_item_html_backfill_filesplit_flow.py

```diff
@@ -0,0 +1,98 @@
+import base64
+import gzip
+import zlib
+from datetime import timedelta
+from io import BytesIO
+from typing import List
+
+import boto3
+import pandas as pd
+import prefect
+from prefect import Flow, task, Parameter, flatten
+from prefect.tasks.aws import S3List, S3Download, S3Upload
+from prefect.executors import LocalDaskExecutor
+
+from utils import config
+from utils.flow import get_flow_name
+
+# Setting flow variables
+FLOW_NAME = get_flow_name(__file__)
+
+# This bucket was created by another process. We may have to revisit using this bucket.
+SOURCE_S3_BUCKET = 'pocket-snowflake-staging-manual'
+SOURCE_PREFIX = 'aurora/textparser-prod-content-snapshot-2022091408-cluster/content/'
+STAGE_S3_BUCKET = 'pocket-data-items'
+STAGE_S3_PREFIX = 'article/backfill-html-filesplit/'
+STAGE_CHUNK_ROWS = 10000
+NUM_FILES_PER_RUN = 1000
+
+
+@task()
+def get_source_keys() -> [str]:
+    """
+    :return: List of S3 keys for the S3_BUCKET and SOURCE_PREFIX
+    """
+    logger = prefect.context.get("logger")
+
+    keys = S3List().run(bucket=SOURCE_S3_BUCKET, prefix=SOURCE_PREFIX)
```

Comment:
You later in this flow you are passing dataframes between tasks. This will load all data matching this prefix into python memory. You can only process as much data in a flow that you can fit it all into memory.

Reply:
@jeshuaborges good point. 
I was thinking about reducing the count on `NUM_FILES_PER_RUN`.
but then, the main purpose of this flow is to only split the files into smaller sized and not take excessive time for processing.
What are your thoughts about not using mappers at all for this flow?